input/pagecontent/changes.md

```diff
@@ -15,4 +15,11 @@ This change log documents the significant updates and resolutions implemented fr
   - Procedure.bodySite and added profile-specific implementation guidance on including body site information [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017).
 - Removed AU Core MedicationStatement and added to Future of AU Core that AU Core MedicationStatement is planned for AU Core R2 [FHIR-45052](https://jira.hl7.org/browse/FHIR-45052).
 - Updated mapping of AUCDI Procedure Completed Event to be represented by Procedure only [au-fhir-core #147](https://github.com/hl7au/au-fhir-core/issues/147).
-- Updated mapping of AUCDI Medication Use Statement to be represented by MedicationStatement only [au-fhir-core #145](https://github.com/hl7au/au-fhir-core/issues/145).
\ No newline at end of file
+- Updated mapping of AUCDI Medication Use Statement to be represented by MedicationStatement only [au-fhir-core #145](https://github.com/hl7au/au-fhir-core/issues/145).
+- Updated AU Core Medication name [FHIR-45032](https://jira.hl7.org/browse/FHIR-45032)
```

Comment:
Remove the entries for the following non-substantive changes
- Updated AU Core Medication name [FHIR-45032](https://jira.hl7.org/browse/FHIR-45032)
- Deleted erroneous must support file [FHIR-45100](https://jira.hl7.org/browse/FHIR-45100)
- Fixed typos on General Requirements page [FHIR-45149](https://jira.hl7.org/browse/FHIR-45149)
- Corrected the wording of STU Note [FHIR-45156](https://jira.hl7.org/browse/FHIR-45156)
- Updated the resource type in the AU Core Waist Circumference example [FHIR-45027](https://jira.hl7.org/browse/FHIR-45027)
- Changed 'supply' to 'supplying' to clarify terminology support guidelines [FHIR-45105](https://jira.hl7.org/browse/FHIR-45105)

Reply:
Done

Reply:
The requested removals were done. In addition an extra removal on the UCUM terminology page change was undertaken. In consideration that this additional removal is non-substantive this will close. It may be requested to be added in later.